src/player.cpp

```diff
@@ -7804,7 +7839,7 @@ void player::suffer()
                     break;
                 case 9:
                     add_msg(m_bad, _("You have the sudden urge to SCREAM!"));
-                    sounds::sound( pos(), 10 + 2 * str_cur, "AHHHHHHH!");
+                    shout("AHHHHHHH!");
```

Comment:
Not your bug, but this should be translated (via `_()`), as are all the other screams.